config/ocp-observatorium-test.yaml

```diff
@@ -29,107 +29,186 @@ loki:
 
 scenarios:
   highVolumeReads:
-    enabled: true
+    enabled: false
     configurations:
-      - description: "Read 20 queries-per-second"
+      - description: "Query range 1 second"
         samples:
-          interval: "15s"
-          range: "1m"
+          interval: "30s"
+          range: "300s"
           total: 10
         writers:
           replicas: 10
-          args:
-            logps: 100
         readers:
           replicas: 1
           throughput: 10
           queries:
-            sumRateByLevel: 'sum(rate({component!=""}[5m])) by (level)'
-          startThreshold: 1024000
-      - description: "Read 20*3 queries-per-second"
+            sumRateByLevel: 'sum(rate({component!=""}[1s])) by (level)'
+            sumRateErrosOnly: 'sum(rate({component!="", level="error"}[1s]))'
+          startThreshold: 1000000
+      - description: "Query range 1 minute"
         samples:
-          interval: "15s"
-          range: "1m"
+          interval: "30s"
+          range: "300s"
           total: 10
         writers:
           replicas: 10
-          args:
-            logps: 100
         readers:
           replicas: 1
           throughput: 10
           queries:
-            topTenComponents: 'topk(10, sum(rate({component!=""}[5m])) by (level))'
-            countOverTime: 'count_over_time({component!=""}[5m])'
-            bytesOverTime: 'bytes_over_time({component!=""}[5m])'
-          startThreshold: 1024000
-      - description: "Read 20*8 queries-per-second"
+            sumRateByLevel: 'sum(rate({component!=""}[1m])) by (level)'
+            sumRateErrosOnly: 'sum(rate({component!="", level="error"}[1m]))'
+          startThreshold: 1500000
```

Comment:
@Red-GV why do we have all those different "startThreshold" values 1000000, 1024000, 1500000 .... maybe we can agree on a common amount of writes before starting the reads ???

Reply:
My initial idea hear was to have different start values for tests that were going to query for more data (such as the week+ long queries). Not sure how much sense it makes it practicality though.